### ISSUE TITLE
VictoryAxis is not aligning tick labels correctly

### ISSUE THREAD

**Issue by johanolandero** [FormidableLabs/victory-native#138]
Greetings,

Numbers on the dependent axis is not aligning properly as shown below. Problem does occur with letters aswell.

![victory-axis](https://user-images.githubusercontent.com/23318359/28724697-3cb283ac-73bb-11e7-877c-63708ffdfe14.png)

```
<VictoryBar
    data={this.props.data}
    x="date"
    y="value"
/>
<VictoryAxis />
<VictoryAxis dependentAxis tickFormat={x => `${x}`} />

**Comment by nicotroia** [comment]
Having the same issue, it seems to just be the `1` for me... all other numbers are fine

<img width="46" alt="screen shot 2017-08-02 at 9 11 45 pm" src="https://user-images.githubusercontent.com/1019948/28901596-42d749e2-77c7-11e7-97f9-c103dac5b9b5.png">
<img width="39" alt="screen shot 2017-08-02 at 9 11 53 pm" src="https://user-images.githubusercontent.com/1019948/28901601-46c9dc04-77c7-11e7-8bb4-e7fa43227a0b.png">

**Comment by johanolandero** [comment]
Work around by setting a custom tickLabelComponent with the prop textAnchor set to "start"

` 
<VictoryAxis
    dependentAxis
    tickLabelComponent={<VictoryLabel textAnchor="start" />}
/>`

**Comment by boygirl** [comment]
this issue is resolved in `victory-native@~0.16.0` + `react-native-svg@^6.0.0`
